modules/olmv1-finding-operators-to-install.adoc

```diff
@@ -7,62 +7,43 @@
 [id="olmv1-finding-operators-to-install_{context}"]
 = Finding Operators to install from a catalog
 
-After you add a catalog to your cluster, you can query the catalog to find Operators and extensions to install. Before you can query catalogs, you must port forward the catalog server service.
+After you add a catalog to your cluster, you can query the catalog to find Operators and extensions to install.
+
+include::snippets/olmv1-on-cluster-catalog-queries.adoc[]
 
 .Prerequisites
 
 * You have added a catalog to your cluster.
 * You have installed the `jq` CLI tool.
+* You have installed the `opm` CLI tool.
 
 .Procedure
 
-. Port forward the catalog server service in the `openshift-catalogd` namespace by running the following command:
-+
-[source,terminal]
-----
-$ oc -n openshift-catalogd port-forward svc/catalogd-catalogserver 8080:443
-----
-
-. In a new terminal window or tab, download the catalog's JSON file locally by running the following command:
+. To return a list of extensions that support the `AllNamespaces` install mode and do not use webhooks, enter the following command:
```

Comment:
Operator Framework dev team: 
Is it possible to expand this query to remove extensions with dependencies?